src/data/docs/documentation.md

```diff
@@ -5,257 +5,232 @@ description: How to write Unison documentation and associate it with your code
+    * buffalo
+3. buffalo.
+}}
 ```
+As you can see, you can intermix your numbered and bullet pointed lists as you please and nest lists arbitrarily.
 
-This is also a [transclusion](https://en.wikipedia.org/wiki/Transclusion). If the type of `myFunction` ever changes (i.e. the name `myFunction` changes so that it refers to a function with a different type), your `Doc` changes automatically as well.
+### Links
+Hyperlinking is supported to external urls, but you can also link to other relevant parts of your codebase via term or type linking in docs blocks. If we were to write `some text which looks [like this.]({type MyType})` we are creating a named type link. Similarly, we might created a named *term* link with the same format: `I am a [term link]({myTermHere})`. Alternatively, if naming our link isn't important to our doc's flow, we can simply embed a type or term link `by writing a term link like so: {someTerm} or a type link like this: {type SomeType}`. Using term and type links within a doc is a handy way to link to other documents. A user might then use the click-through capabilities in an interactive renderer of the docs to understand the relationship between code components or learn more through relevant urls. 
 
-### Include the full contents of another Doc value
+# Suggested conventions
 
-To nest a `Doc` value inside another `Doc` value, use `@[include]` followed by
-a name given to that `Doc` value in your codebase. When Unison displays the `Doc`, the entire contents of the included `Doc` will be displayed in that spot.
+Although documentation values don't require any particular structure you might try structuring your docs in accordance with the following guidelines: 
 
-The rule about [transclusion](https://en.wikipedia.org/wiki/Transclusion) applies here too; if the referenced `Doc` value changes, your `Doc` changes automatically as a result.
-
-Note that Unison will not let you create a cycle where you include a `Doc` inside of itself.
-
-### Include any Unison value from the codebase
-
-Sometimes you want to include the result of evaluating some Unison expression. To do this, use `@[evaluate]` followed by the name of a value in your codebase. For example, given a definition like this:
-
-```unison
-exampleExpression = 2 + 2
-```
-
-We'll have to first add it to the codebase:
-
-```ucm
-.> add
-
-  ⍟ I've added these definitions:
-
-    exampleExpression : Nat
-```
-
-Once the definition is in the codebase, we can insert the result of evaluating it
-into any `Doc` value:
-
-```unison
-exampleDoc =
-  [: The value is @[evaluate] exampleExpression :]
-```
-
-When Unison displays `exampleDoc`, this will get expanded to:
+- Start with a brief overview, then maybe a longer description, then some examples, then link to related definitions and further reading.
+- Use Markdown syntax for formatting. Indent examples by 4 spaces for readability when viewing the docs in plain text form, and this will also render nicely as Markdown.
```

Comment:
I don't understand "indent examples by 4 spaces". Does this mean code? I feel like this may need an example.